RecoCTPPS/TotemRPLocal/python/totemRPLocalReconstruction_cff.py

```diff
@@ -0,0 +1,24 @@
+import FWCore.ParameterSet.Config as cms
+
+# geometry
+from Geometry.VeryForwardGeometry.geometryRP_cfi import *
+XMLIdealGeometryESSource.geomXMLFiles.append("Geometry/VeryForwardData/data/RP_Garage/RP_Dist_Beam_Cent.xml")
```

Comment:
@ianna 
yes, we need this code running in the standard sequences.
Please point @jan-kaspar to a relevant file(s) to be changed to include the geometry setup.